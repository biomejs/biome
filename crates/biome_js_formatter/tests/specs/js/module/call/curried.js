foo((argument1) =>
    (argument2) =>
    (argument3) =>
    (argument4) =>
    (argument5) =>
    (argument6) =>
    (argument7) =>
    (argument8) =>
    (argument9) =>
    (argument10) =>
    (argument11) =>
    (argument12) =>3);

foo((argument1) =>
    (argument2) =>
    (argument3) =>
    (argument4) =>
    (argument5) =>
    (argument6) =>
    (argument7) =>
    (argument8) =>
    (argument9) =>
    (argument10) =>
    (argument11) =>
    (argument12) => ({
        foo: bar,
        bar: baz,
        baz: foo
    }));