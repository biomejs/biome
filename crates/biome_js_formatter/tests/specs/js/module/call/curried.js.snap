---
source: crates/biome_formatter_test/src/snapshot_builder.rs
info: js/module/call/curried.js
---

# Input

```js
foo((argument1) =>
    (argument2) =>
    (argument3) =>
    (argument4) =>
    (argument5) =>
    (argument6) =>
    (argument7) =>
    (argument8) =>
    (argument9) =>
    (argument10) =>
    (argument11) =>
    (argument12) =>3);

foo((argument1) =>
    (argument2) =>
    (argument3) =>
    (argument4) =>
    (argument5) =>
    (argument6) =>
    (argument7) =>
    (argument8) =>
    (argument9) =>
    (argument10) =>
    (argument11) =>
    (argument12) => ({
        foo: bar,
        bar: baz,
        baz: foo
    }));
```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Quote style: Double Quotes
JSX quote style: Double Quotes
Quote properties: As needed
Trailing comma: All
Semicolons: Always
Arrow parentheses: Always
Bracket spacing: true
Bracket same line: false
-----

```js
foo(
	(argument1) =>
		(argument2) =>
		(argument3) =>
		(argument4) =>
		(argument5) =>
		(argument6) =>
		(argument7) =>
		(argument8) =>
		(argument9) =>
		(argument10) =>
		(argument11) =>
		(argument12) =>
			3,
);

foo((argument1) =>
	(argument2) =>
	(argument3) =>
	(argument4) =>
	(argument5) =>
	(argument6) =>
	(argument7) =>
	(argument8) =>
	(argument9) =>
	(argument10) =>
	(argument11) =>
	(argument12) => ({
		foo: bar,
		bar: baz,
		baz: foo,
	}));
```


